proposals/devcontainer-features-distribution.md

```diff
@@ -120,7 +120,7 @@ NAMESPACE=devcontainers/features
 
 oras push ${REGISTRY}/${NAMESPACE}:latest \
         --manifest-config /dev/null:application/vnd.devcontainers \
-                            ./devcontainer-collection.json.:application/vnd.devcontainers.layer.v1+json
+                            ./devcontainer-collection.json:application/vnd.devcontainers.collection.layer.v1+json
```

Comment:
Curious, looking at the `Manifest` in [here](https://github.com/devcontainers/features/pkgs/container/features/32903232?tag=latest), where does the `collection.layer` part come from?

Reply:
That github rendering is not actually accurate to what I see if I inspect it manually with a tool like `skopeo`